pkg/registry/helpers.go

```diff
@@ -1,8 +1,6 @@
 package registry
 
-import (
```

Comment:
I feel like I am not processing some idiomatic Go stuff. would love to learn more (to avoid this)